src/passes/Inlining.cpp

```diff
@@ -588,6 +588,14 @@ struct FunctionSplitter {
       // return), and we would not even attempt to do splitting.
       assert(body->is<Block>());
 
+      Index outlinedFunctionSize =
+        info.size - Measurer::measure(iff->condition) - /* if itself */ 1;
```

Comment:
```suggestion
        info.size - Measurer::measure(iff);
```
This should be identical, and is simpler. Was the current code an optimization to save work? If so I think it's too small to be worth optimizing. (In particular, a risk is that the measure metric could be improved in the future, so this math might end up wrong.)

Reply:
This wasn't intended as an optimization; was trying to avoid the "false" body since I assume this pattern could "technically" work with an "else" as well. 
However since absence of "else" is enforced, I can simplify it.